kit/media/src/UIVideo/UIVideo.web.tsx

```diff
@@ -6,12 +6,38 @@ import type { UIVideoProps } from './types';
 export const UIVideo: React.FC<UIVideoProps> = ({
     uri,
     controls,
+    paused = false,
+    muted,
     repeat,
     width,
     height,
     aspectRatio,
+    resizeMode = 'contain',
 }: UIVideoProps) => {
     const dimensions = useDimensionsByAspectRatio(width, height, aspectRatio);
 
-    return <ReactPlayer url={uri} controls={controls} {...dimensions} loop={repeat} />;
+    return (
+        <ReactPlayer
+            url={uri}
+            controls={controls}
+            {...dimensions}
+            loop={repeat}
+            playing={!paused}
+            muted={muted}
+            /**
+             * For apply resizeMode
+             */
+            config={{
```

Comment:
Do you think if we should apply `React.useMemo` for it, or we can apply `React.memo` for the whole component? to decrease unneeded re-renders

Reply:
Both 😄